test/test_bedclassifier.py

```diff
@@ -0,0 +1,56 @@
+import os
+from tempfile import TemporaryDirectory
+
+from bedboss.bedclassifier import BedClassifier, get_bed_type
+
+
+FILE_DIR = os.path.dirname(os.path.realpath(__file__))
+HG19_CORRECT_DIR = os.path.join(FILE_DIR, "test_data", "bed", "hg19", "correct")
+FILE_PATH = f"{HG19_CORRECT_DIR}/sample1.bed.gz"
+FILE_PATH_UNZIPPED = f"{HG19_CORRECT_DIR}/hg19_example1.bed"
+
+
+def test_classification():
+    with TemporaryDirectory() as d:
+        bedclass = BedClassifier(input_file=FILE_PATH, output_dir=d)
+        print("DEBUG BEDCLASS\n")
+        print(bedclass.bed_type)
+
+
+def test_get_bed_type():
+    bedtype = get_bed_type(bed=FILE_PATH_UNZIPPED)
+    print("DEBUG BEDTYPE\n")
+    print(bedtype)
+
```

Comment:
Before merging, please delete all prints and comments, or set skip, to identify it as manual tests

Reply:
I added skip to some of them. I'm leaving the commented out manual tests as I have a feeling I will need to iterate on the be classification in the near future.